src/main/content/_assets/js/toc.js

```diff
@@ -78,6 +78,10 @@ function updateTOCHighlighting(id) {
     $('.liSelected').removeClass('liSelected');
     var anchor = $("#toc_container a[href='#" + id + "']");
     anchor.parent().addClass('liSelected');
+    var targetElm = document.querySelector('.liSelected');
+    if(targetElm) {
```

Comment:
```suggestion
    if (targetElm) {
```

To have consistent syntax styling in the file if we every need to do mass `grep` on a file.

Reply:
ok